doc/source/language/delegates.rst

```diff
@@ -402,5 +403,5 @@ can't be changed easily.*)
     print "altpos at altitude 1000 is " + altpos:call(1000).
 
   However, if you like you can make your own user function that is a
-  wrapper around a structure suffix call, and make a delegate of THAT.
+  wrapper around a structure suffix call, and make a delegate of *that*.
```

Comment:
@hvacengi There are too many standards. AP, Chicago, Strunk & White, etc. I don't know that kOS documentation is a formal enough project to officially adopt a style guide, so I'm just going with some commonsense guesses. One source I like, especially for modern, online text is *Butterick's Practical Typography*. [Here it is on bold and italic.][bpt]

BPT (and most sources) say italic is emphasis and bold is stronger emphasis. BPT notes that italic is often poorly rendered or hard to read on-screen, so it may be better to go straight to bold for online text. All-caps is sometimes described as an emphasis option, but also has the unpleasant “shouting” connotation.

In any case, it seems that bold is a good consensus choice here. I'll be happy to change it.

[bpt]: http://practicaltypography.com/bold-or-italic.html "Bold or italic | Butterick's Practical Typography"